tap_kustomer/error.py

```diff
@@ -1,6 +1,6 @@
 class AssertionException(Exception):
     def __init__(self, message):
-        super(AssertionException, self).__init__(message)
+        super().__init__(message)
```

Comment:
According to this stackoverflow post and pep referenced:
https://stackoverflow.com/questions/10482953/python-extending-with-using-super-python-3-vs-python-2
http://www.python.org/dev/peps/pep-3135/
super() in python3 is equivalent to super(class, self)
fixed cause pylint warned about it